backend/routes/auth/authorize.py

```diff
@@ -21,7 +27,47 @@ class AuthorizeRequest(BaseModel):
 
 
 class AuthorizeResponse(BaseModel):
-    token: str = Field(description="A JWT token containing the user information")
+    username: str = Field("Discord display name.")
+    expiry: str = Field("ISO formatted timestamp of expiry.")
+
+
+AUTH_FAILURE = JSONResponse({"error": "auth_failure"}, status_code=400)
```

Comment:
This should be above AuthorizeResponse and AuthorizeRequest.

Reply:
Addressed in ca73008.